### PR TITLE
refactor: abstract Raylib keyboard managment

### PR DESCRIPTION
**Pull request by barbo69**
We don't need to indicate the type of input and consequently we dont need to rename Keyboard Component to Event Component

### CODE REVIEW

**Review by lucas-louis — APPROVED** [review]
LGTM

File: OtterLib/graphic/include/OtterGraphicKeyboard.hpp [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+         * @brief The key enum
+         * @details The key enum
+         * @enum Key::LEFT: The left key
+         * @enum Key::RIGHT: The right key
+         * @enum Key::UP: The up key
+         * @enum Key::DOWN: The down key
+         * @enum Key::SPACE: The space key
+         * @enum Key::ESCAPE: The escape key
+         */
+        enum KeyType {
```
Comment (fzuefo) [inline OtterLib/graphic/include/OtterGraphicKeyboard.hpp:37]: need to verifie if this not destroy virtuality of the class

**Review by fzuefo — CHANGES_REQUESTED** [review]
the change are fine to me, but pls solve conflic merge before

File: OtterLib/network/include/NetChannel.hpp [hunk 2]
```diff
@@ -1,32 +1,32 @@
-#pragma once
+// #pragma once
 
-#include "asio.hpp"
+// #include "asio.hpp"
```
[the remark this answers is not in the document]
Comment (fzuefo) [reply OtterLib/network/include/NetChannel.hpp:3]: not deprecate code but non fonctional part

**Review by fzuefo — APPROVED** [review]

**Pull request merged**